arangod/Aql/ClusterQuery.cpp

```diff
@@ -60,12 +60,10 @@ ClusterQuery::ClusterQuery(QueryId id,
             std::move(options),
             /*sharedState*/ ServerState::instance()->isDBServer()
                 ? nullptr
-                : std::make_shared<SharedQueryState>(ctx->vocbase().server())},
-      _planMemoryUsage(0) {}
```

Comment:
note: the `_planMemoryUsage` member is now in base class `Query`, because it can used for ClusterQuery and non-ClusterQuery Querys alike.